bigbluebutton-html5/imports/ui/components/video-provider/service.js

```diff
@@ -134,6 +135,11 @@ class VideoService {
     return m.lockSettingsProps ? m.lockSettingsProps.disableCam : false;
   }
 
+  mirrorOwnWebcam(user) {
+    // only true if setting defined and video ids match
+    return getFromUserSettings('bbb_mirror_own_webcam', Meteor.settings.public.app.mirrorOwnWebcam) && this.userId() === user.userId;
```

Comment:
Could you pass this Meteor setting to a constant in the top of the file like [this](https://github.com/bigbluebutton/bigbluebutton/pull/9828/files#diff-200c74915e2353b2e968e7f72bd7712bR11)